sdk-lib/src/main/java/cash/z/ecc/android/sdk/tool/DerivationTool.kt

```diff
@@ -82,6 +82,41 @@ interface DerivationTool {
         network: ZcashNetwork
     ): String
 
+    /**
+     * Derives a [ZIP 32 Arbitrary Key] from the given seed at the "wallet level", i.e.
+     * directly from the seed with no ZIP 32 path applied.
+     *
+     * The resulting key will be the same across all networks (Zcash mainnet, Zcash
+     * testnet, OtherCoin mainnet, and so on). You can think of it as a context-specific
+     * seed fingerprint that can be used as (static) key material.
+     *
+     * [ZIP 32 Arbitrary Key]: https://zips.z.cash/zip-0032#specification-arbitrary-key-derivation
+     *
+     * @param contextString a globally-unique non-empty sequence of at most 252 bytes that
+     *        identifies the desired context.
```

Comment:
If failing to meet this requirement can cause a panic, it must also be documented at this layer.